### PR TITLE
Telemetry logging updated fat

### PR DESCRIPTION
**Pull request by tonyreigns**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

Related to #27711

### CODE REVIEW

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryApplicationConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+    private static Class<?> c = TelemetryApplicationConfigTest.class;
+
+    public static final String APP_NAME = "MpTelemetryLogApp";
+    public static final String SERVER_NAME = "TelemetryAppConfig";
+
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    public static final String SERVER_XML_ALL_SOURCES = "allSources.xml";
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryApplicationConfigTest.java]: I don't think these are used anywhere in this class.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+
+@RunWith(FATRunner.class)
+public class TelemetryDropinsTest extends FATServletClient {
+
+    private static Class<?> c = TelemetryDropinsTest.class;
+
+    public static final String APP_NAME = "MpTelemetryLogApp";
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java:39]: Not used.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCTest.java [hunk 3]
```diff
@@ -92,6 +92,7 @@ public void testTelemetryFFDCMessages() throws Exception {
 
     static void testTelemetryFFDCMessages(LibertyServer s, Consumer<List<String>> consoleConsumer) throws Exception {
         hitWebPage(s, "ffdc-servlet", "FFDCServlet", true, "?generateFFDC=true");
+        Thread.sleep(2000);
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCTest.java]: Is this still needed?

**Review by pgunapal — CHANGES_REQUESTED** [review]
looks good, minor comments

**Comment by tonyreigns** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    }
+
+    /*
+     * Test an application exported to dropins with an application SDK and ensure only application logs are bridged.
+     */
+    @Test
+    public void testTelemetryDropinAppSDK() throws Exception {
+        server.startServer();
+
+        WebArchive app = ShrinkWrap
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java:72]: should we wait till the server is started before dropping in?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    /*
+     * Test an application exported to dropins with a Runtime SDK and ensure both application and runtime logs are bridged.
+     */
+    @Test
+    public void testTelemetryDropinRuntimeSDK() throws Exception {
+        server.addEnvVar("OTEL_LOGS_EXPORTER", "logging");
+        server.addEnvVar("OTEL_SDK_DISABLED", "false");
+        server.startServer();
+
+        WebArchive app = ShrinkHelper.buildDefaultApp(APP_NAME, "io.openliberty.microprofile.telemetry.logging.internal.fat.MpTelemetryLogApp");
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java:115]: shouldn't we wait till server is started?

**Review by pgunapal — CHANGES_REQUESTED** [review]

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java (hunk 4 above)
Comment (pgunapal) [reply dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java:72]: Will be covered in a follow on PR.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java (hunk 5 above)
Comment (pgunapal) [reply dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryDropinsTest.java:115]: Will be covered in a follow on PR.

**Review by pgunapal — APPROVED** [review]
Looks good, some minor TO-DOs to be added in a follow on PR.

**Pull request merged**